synapse/replication/http/streams.py

```diff
@@ -25,18 +25,31 @@
 class ReplicationGetStreamUpdates(ReplicationEndpoint):
     """Fetches stream updates from a server. Used for streams not persisted to
     the database, e.g. typing notifications.
+
+    The API looks like:
+
+        GET /_synapse/replication/get_repl_stream_updates/events?from_token=0&to_token=10&limit=100
```

Comment:
`from_token` is inclusive, `to_token` is exclusive?

Reply:
I thought we generally avoided saying inclusive vs exclusive, and rather thought about tokens as being "between" items? I don't know if that is helpful, can add the comment in